snap/snapcraft.yaml

```diff
@@ -144,6 +144,8 @@ parts:
       - cmake
       - curl
       - libgtk-3-dev
+      - libgstreamer1.0-dev
+      - libgstreamer-plugins-base1.0-dev
```

Comment:
could we move these as build-packages of the ubuntu-desktop-installer part so they won't go missing if we get to replace flutter-git with the flutter extension?